app/services/admin/flags/ban_user.rb

```diff
@@ -28,7 +26,15 @@ def success?
 
       private
 
-      attr_reader :flag
+      attr_reader :flag, :admin
+
+      def update_flag_for_ban(flag)
```

Comment:
`flag` will be available to this method through the `attr_reader` so we can remove this argument :)

Reply:
ahh yes that makes sense. thanks